manuscript/100_Analysis_In_TDD.md

```diff
@@ -42,7 +42,7 @@ Gherkin
 
 Hungry? Too bad, because the Gkerkin I�am gonna talk about is not edible. It is a�notation and a�way of thinking about behaviors of the specified piece of code. It can be applied on different levels of granularity -- any behavior, whether of a�whole system or a�single class, may be described using it.
 
-We actually talked about Gherkin before in this book, just did not name it. It is the GIVEN-WHEN-THEN structure that you can see everywhere, even in code samples as comments. This time, we are stamping a�name on it and analyzing it further.
+We talked about Gherkin before in this book, just did not name it. It is the GIVEN-WHEN-THEN structure that you can see everywhere, even in code samples as comments. This time, we are stamping a�name on it and analyzing it further.
```

Comment:
removing "actually" made me think hard about what I wanted to say here and figured out that it was something like this: "although I'm mentioning Gherkin now, we already talked about it." and the "actually" was used to address a presumed surprise of the reader. What is the idiom to better state such things?